app/controllers/api/cloud_networks_controller.rb

```diff
@@ -9,5 +9,17 @@ def delete_resource(type, resource_id, _data = {})
       task_id = cloud_network.delete_cloud_network_queue(User.current_user.userid)
       action_result(true, "Deleting #{cloud_network.name}", :task_id => task_id)
     end
+
+    def options
+      return super unless params.keys.include_any?(%w[id ems_id])
+
+      if params.key?("ems_id")
+        ems = ExtManagementSystem.find(params["ems_id"])
```

Comment:
Does this need RBAC? (same with the find call below.

Reply:
Yes need to convert over to resource_search just nailing down the type / collection_type params since they're not passed in

Reply:
This is also just an example, if we're okay with the routes match I'll drop this commit from the PR since it has its own PR https://github.com/ManageIQ/manageiq-api/pull/1058